### PR TITLE
KUR-39: Add test coverage for the livelogpost lists service and make …

### PR DESCRIPTION
**Pull request by fago**
…the module install on a fresh installation.

### CODE REVIEW

File: tests/src/Functional/LiveBlogListTest.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+use Drupal\Component\Serialization\Json;
+use Drupal\liveblog\Entity\LiveblogPost;
+use Drupal\node\Entity\Node;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Functional tests for the liveblog list.
+ *
+ * @group serialization
```
Comment (fago) [inline tests/src/Functional/LiveBlogListTest.php]: No liveblog.

File: config/install/field.storage.node.field_status.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+uuid: 76e6e42b-0139-475b-9365-7e63c154a9e3
```
Comment (fago) [inline config/install/field.storage.node.field_status.yml]: true, maybe it's auto-removed. removed.

File: tests/src/Functional/LiveBlogListTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      'title' => 'Title post 1',
+      'body' => 'Body post 1',
+    ])->save();
+  }
+
+  /**
+   * Tests using the liveblog-list route.
+   */
+  public function testLiveBlogListRoute() {
+
```
Comment (fago) [inline tests/src/Functional/LiveBlogListTest.php]: This empty line is part of the coding standard. After each method comes an empty line, check cor.

**Comment by fago** [comment]
I addressed your points, except the last one.

**Comment by milkovsky** [comment]
looks good. let's merge

**Pull request merged**